doc-content/enterprise-only/admin-and-config/define-role-based-access-proc.adoc

```diff
@@ -7,11 +7,11 @@ You can customize contributor role permissions for each branch of a project in {
 . If needed, add a new contributor:
 .. Click the project name and then click the *Contributors* tab.
 .. Click *Add Contributor*.
-.. Select the role name from the *admin* pull-down menu, for example, *analyst*.
-.. Select the *Contributor* role type from the *Owner* pull-down menu.
+.. Enter a user name in the text field.
```

Comment:
Suggested rewrite:
Enter user name in the...

Reply:
@hmanwani-rh, I added the change.